File: Golang/Bot.go
package main

import ("log" "github.com/go-telegram-bot-api/telegram-bot-api")

func main() {
    bot, err := tgbotapi.NewBotAPI("585665:AA")
    
   if err != nil {
      log.Panic(err)
    }

    bot.Debug = true
    log.Printf("Authorized on account %s", bot.Self.UserName)

    u := tgbotapi.NewUpdate(0)
    u.Timeout = 60
    updates, err := bot.GetUpdatesChan(u)

    for update := range updates {
        if update.Message == nil { 
            continue
        }

        if update.Message.Text == "/start" {
            msg := tgbotapi.NewMessage(update.Message.Chat.ID, "Hi ayhan - @Ayhan_Dev")
            _, err := bot.Send(msg)
            if err != nil {
                log.Println(err)
            }
        }
    }
}

File: Php/Bot.php
<?php

include 'lib.php';

$token = "6195507430:AAEQAVfqacaizHINgCYkVGPEQ5MQWJ_tuR8";

try {
    class las_bot extends Bot {
        const TOKEN = $token;

        public function onUpdate($update) {
            $text = $update['message']['text'];
            if ($text === "/start") {
                $this->sendMessage([
                    'chat_id' => $update['message']['chat']['id'],
                    'text' => 'hi my bibi :) @ayhan_dev',
                ]);
            }
        }
    }

    $bot = new las_bot($token);
} catch (Exception $e) {
    echo $e->getMessage();
}
?>
